ui/src/app/collection/detail/manage-collection.component.html

```diff
@@ -139,7 +139,7 @@ <h2 class="t-type-heading1 t-margin-small t-margin-bottom">Confirm that you want
       <div class="t-margin-medium t-margin-bottom">
         <div class="l-container l-container-mobile t-margin-medium t-margin-bottom">
           <h2 class="t-type-heading1 t-margin-small t-margin-bottom">
-            Confirm that you want to delete {{this.collection?.name}}
+            Confirm that you want to {{collection.status === "Workspace" ? "reset" : "delete"}} {{this.collection?.name}}
```

Comment:
Since this is duplication of code, it makes sense to add it as a function in the component.

Reply:
Now there are confirmMessageText() and confirmButtonText()
These functions are in manage-collection.component and are overridden in my-workspace.component.